tests/x11/keyboard_layout_gnome.pm

```diff
@@ -0,0 +1,80 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+
+# Summary: Test if gnome login manager honors keyboard layout changes
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub restart_x11 {
+    my ($self, %args) = @_;
+    $args{setnologin} //= 0;
+    record_soft_failure('bsc#1063060 workaround for graphical errors on cirrus when restarting X11 -> performing reboot');
```

Comment:
1. unconditional soft_failures are bad. Can you check somehow that bug actually there ?
2. In comment 2 in bug it stated that better way just to `systemctl restart display-manager` why not just do it this way ?

Reply:
> 1. unconditional soft_failures are bad. Can you check somehow that bug actually there?

Not without breaking the test. Also this is already done the same way in multi_user_dm.

> 2. In comment 2 in bug it stated that better way just to systemctl restart display-manager why not just do it this way?

A reboot might be the better choice for this test, anyway.